src/server/pps/server/worker_rc.go

```diff
@@ -153,11 +154,11 @@ func (kd *kubeDriver) workerPodSpec(options *workerOptions, pipelineInfo *pps.Pi
 		Name:  client.PPSPipelineNameEnv,
 		Value: pipelineInfo.Pipeline.Name,
 	}, {
-		Name:  "LOKI_SERVICE_HOST_VAR",
-		Value: kd.config.LokiHostVar,
+		Name:  "LOKI_SERVICE_HOST",
+		Value: kd.config.LokiHost,
 	}, {
-		Name:  "LOKI_SERVICE_PORT_VAR",
-		Value: kd.config.LokiPortVar,
+		Name:  "LOKI_SERVICE_PORT",
+		Value: kd.config.LokiPort,
```

Comment:
I think this is fine but be aware that we change the semantics here.  Before, LOKI_SERVICE_HOST_VAR (etc.) was set at RC creation time, and each pachyderm replica being created by the RC would look up the environment variable at startup time (since pachd did the resolution, not anything in k8s).  

Typically the variable this was pointing at was a variable based on services in the cluster (a normal service named `foo` causes FOO_SERVICE_... variables to be set at pod creation time, by the kubelet).  What this means is that each replica could have its own unique value if you edited the service between replica creations; i.e. it was 1.2.3.4:1234 for the first replica, then you edit the service to point at 10.0.0.123:1234, then a day passes and someone submits two jobs at the same time and autoscaling is turned on, so the second replica gets 10.0.0.123:1234 as the loki address.

With your change, this doesn't happen.  pachd looks this up at startup, and applies the same value to every RC it creates.

The old behavior is honestly confusing and hard to debug, since we go our of our way to make a new RC when the environment changes (or actually, I think whenever a new PPS master is elected/created), and this is better.  But, it is a change!

Reply:
That's true. I am not sure how to best call this out, but it might be a good thing to mention in our support docs